invenio_theme/bundles.py

```diff
@@ -52,20 +52,28 @@
 )
 """Default CSS bundle with Almond, Bootstrap, Font-Awesome and JQuery."""
 
-admin_css = NpmBundle(
+admin_lte_css = NpmBundle(
     'node_modules/admin-lte/dist/css/AdminLTE.min.css',
     'node_modules/admin-lte/dist/css/skins/{0}.min.css'.format(
         # current_app.config.get('ADMIN_UI_SKIN')
         'skin-blue'
     ),
     filters='cleancss',
-    output='gen/styles.admin.%(version)s.css',
+    output='gen/styles.admin-lte.%(version)s.css',
     npm={
         'admin-lte': '~2.3.6',
     }
 )
-"""Default style for admin interface."""
 
+admin_scss = NpmBundle(
```

Comment:
Wasn't sure what to call it in fact. I had to separate the css and scss due to compilation issues. So, will I move it to css?